src/sentry/api/endpoints/event_apple_crash_report.py

```diff
@@ -46,15 +46,8 @@ def get(self, request, event_id):
                 'message': 'Only cocoa events can return an apple crash report',
             }, status=403)
 
-        threads = event.data.get(
-            'sentry.interfaces.threads',
-            event.data.get('threads'
-        )).get('values')
-
-        exception = event.data.get(
-            'sentry.interfaces.Exception',
-            event.data.get('exception'
-        )).get('values')
+        threads = (event.data.get('threads') or {}).get('values')
```

Comment:
this code can never actually encounter `sentry.interfaces.threads` which is wrong anyways (if anything it would be `sentry.interfaces.Threads`) or `exception` (which always is `sentry.interfaces.Exception`). So since I already touched that code i wanted to fix that.